Assets/GothicVR/Scripts/Creator/MusicCreator.cs

```diff
@@ -50,6 +57,27 @@ void Start()
             musicSource = backgroundMusic.AddComponent<AudioSource>();
         }
 
+        /// <summary>
+        /// Called once every 0.20 seconds to update the music once the timer is bigger than the updateInterval
+        /// </summary>
+        void FixedUpdate()
+        {
+            if (!FeatureFlags.I.EnableMusic)
+                return;
+
+            if (!WorldCreator.I.IsWorldLoaded)
+            {
+                UpdateMusic();
+                return;
+            }
+            timer += Time.fixedDeltaTime;
+            if (timer >= updateInterval)
```

Comment:
Please give the variable a more meaningful name. How about sth. like _changeMusicAfter_ etc.?